build.gradle

```diff
@@ -84,10 +85,10 @@ dependencies {
     implementation group: 'ch.qos.logback.contrib', name: 'logback-jackson', version: '0.1.5'
 
     // OpenTelemetry dependencies:
-    // Spring Boot 3.4.0 pulls in opentelemetry-bom 1.43.0.
-    // We choose opentelemetry-instrumentation-bom-alpha:2.9.0-alpha because otel-instrumentation 2.9.0 targets otel 1.43.0.
+    // Spring Boot 3.4.2 pulls in opentelemetry-bom 1.43.0.
+    // Note that opentelemetry-instrumentation-bom-alpha:2.12.0-alpha targets opentelemetry 1.46.0.
```

Comment:
@LizBaldo I'm not sure if it's a problem to be diverging here in terms of what the Spring Boot plugin and the opentelemetry bom each define. It was a problem in the past as I recall (hence these comments) but I think that if it builds and tests succeed we're ok.